datadog_checks_base/datadog_checks/base/utils/time.py

```diff
@@ -0,0 +1,73 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import absolute_import
+
+from datetime import datetime
+from time import time as epoch_offset
+
+import pytz
+from six import PY2
+
+UTC = pytz.utc
+EPOCH = datetime(1970, 1, 1, tzinfo=UTC)
+
+
+if PY2:
```

Comment:
Any reason not to use something like this for both py2 and py3 ?

```
def get_timestamp(dt=None):
    """
    Returns the number of seconds since the Unix epoch.
    If `dt` is not specified or `None`, the current time in UTC is assumed.
    """
    if dt is None:
        return epoch_offset()

    return (normalize_datetime(dt) - EPOCH).total_seconds()
```

Reply:
Because you would never do it like that in Python 3, & when we drop Python 2 we would never notice without a code comment

Reply:
Python3 does the same thing:

https://github.com/python/cpython/blob/3.8/Lib/datetime.py#L1788-L1794

And since we normalise to always have a tz, we will never to to this condition `if self._tzinfo is None`.